### PR TITLE
Partial Translation for command-line.asc and guis.asc - 1128 translated words

### PR DESCRIPTION
**Pull request by dranem**
Translated 1128 words from english to tagalog

### CODE REVIEW

File: book/A-git-in-other-environments/sections/guis.asc [hunk 1]
```diff
@@ -1,112 +1,109 @@
 === Graphical Interfaces
 
 (((GUIs)))(((Graphical tools)))
-Git's native environment is in the terminal.
-New features show up there first, and only at the command line is the full power of Git completely at your disposal.
-But plain text isn't the best choice for all tasks; sometimes a visual representation is what you need, and some users are much more comfortable with a point-and-click interface.
+Ang nitibong environment ng Git ay nasa terminal.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: `nitibong` to `natibong`

File: book/A-git-in-other-environments/sections/guis.asc [hunk 2]
```diff
@@ -1,112 +1,109 @@
...
+Ang mga bagong tampok ay ipapakita muna roon, at tanging sa command line ay ang buong lakas ng Git na lubos sa iyong pamamahala.
+Ngunit ang plain text ay hindi ang pinakamahusay na pagpipilian para sa lahat ng mga gawain; kung minsan ang isang biswall na representasyon ay ang iyong kailangan, at ang ilang mga gumagamit ay mas kumportable sa isang interface ng point-and-click.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: `biswall` to `biswal`

File: book/A-git-in-other-environments/sections/guis.asc [hunk 3]
```diff
@@ -1,112 +1,109 @@
...
-When viewed in this light, none of these tools can be called ``better'' than any of the others, they're simply more fit for their intended purpose.
-Also note that there's nothing these graphical clients can do that the command-line client can't; the command-line is still where you'll have the most power and control when working with your repositories.
+Mahalagang tandaan na ang iba't ibang mga interface ay natatangi para sa iba't-ibang mga workflow.
+Ang ilang mga kliyente ay naglalantad lamang ng maingat na na-curate na subset ng Git functionality, upang suportahan ang isang partikular na paraan ng paggawa na itinuturing ng may-akda na epektibo.
+Kapag tiningnan sa liwanag na ito, wala sa mga tool na ito ay maaaring masabi na ``mas mabuti'' kaysa sa iba pa, ang mga ito ay mas angkop sa kanilang layunin.
+Tandaan din na wala sa mga graphical na kliyente na maaaring gawin na ang command-line client ay hindi kaya; ang command-line ay pa rin kung saan magkakaroon ka ng pinaka makapangyarihan at kontrol kapag gumagawa sa iyong mga repository.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: - you could transliterate `graphical` to `grapikal`
- `command-line na kliyente`

File: book/A-git-in-other-environments/sections/guis.asc [hunk 4]
```diff
@@ -1,112 +1,109 @@
...
+Ngunit ang plain text ay hindi ang pinakamahusay na pagpipilian para sa lahat ng mga gawain; kung minsan ang isang biswall na representasyon ay ang iyong kailangan, at ang ilang mga gumagamit ay mas kumportable sa isang interface ng point-and-click.
 
-It's important to note that different interfaces are tailored for different workflows.
-Some clients expose only a carefully curated subset of Git functionality, in order to support a specific way of working that the author considers effective.
-When viewed in this light, none of these tools can be called ``better'' than any of the others, they're simply more fit for their intended purpose.
-Also note that there's nothing these graphical clients can do that the command-line client can't; the command-line is still where you'll have the most power and control when working with your repositories.
+Mahalagang tandaan na ang iba't ibang mga interface ay natatangi para sa iba't-ibang mga workflow.
+Ang ilang mga kliyente ay naglalantad lamang ng maingat na na-curate na subset ng Git functionality, upang suportahan ang isang partikular na paraan ng paggawa na itinuturing ng may-akda na epektibo.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: `Git na functionality`

File: book/A-git-in-other-environments/sections/guis.asc [hunk 5]
```diff
@@ -1,112 +1,109 @@
...
+==== `gitk` at `git-gui`
 
 (((git commands, gitk)))(((git commands, gui)))(((gitk)))
-When you install Git, you also get its visual tools, `gitk` and `git-gui`.
+Kapag nag-install ka ng Git, nakakuha ka rin ng mga visual na tool nito, `gitk` at `git-gui`.
 
-`gitk` is a graphical history viewer.
-Think of it like a powerful GUI shell over `git log` and `git grep`.
-This is the tool to use when you're trying to find something that happened in the past, or visualize your project's history.
+`gitk` ay isang graphical viewer ng kasaysayan.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: ``Ang `gitk` ``

File: book/A-git-in-other-environments/sections/guis.asc [hunk 6]
```diff
@@ -1,112 +1,109 @@
...
+Ang ilang mga kliyente ay naglalantad lamang ng maingat na na-curate na subset ng Git functionality, upang suportahan ang isang partikular na paraan ng paggawa na itinuturing ng may-akda na epektibo.
+Kapag tiningnan sa liwanag na ito, wala sa mga tool na ito ay maaaring masabi na ``mas mabuti'' kaysa sa iba pa, ang mga ito ay mas angkop sa kanilang layunin.
+Tandaan din na wala sa mga graphical na kliyente na maaaring gawin na ang command-line client ay hindi kaya; ang command-line ay pa rin kung saan magkakaroon ka ng pinaka makapangyarihan at kontrol kapag gumagawa sa iyong mga repository.
 
-==== `gitk` and `git-gui`
+==== `gitk` at `git-gui`
 
 (((git commands, gitk)))(((git commands, gui)))(((gitk)))
-When you install Git, you also get its visual tools, `gitk` and `git-gui`.
+Kapag nag-install ka ng Git, nakakuha ka rin ng mga visual na tool nito, `gitk` at `git-gui`.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: `biswal` for consistency

File: book/A-git-in-other-environments/sections/guis.asc [hunk 7]
```diff
@@ -1,112 +1,109 @@
...
-Think of it like a powerful GUI shell over `git log` and `git grep`.
-This is the tool to use when you're trying to find something that happened in the past, or visualize your project's history.
+`gitk` ay isang graphical viewer ng kasaysayan.
+Isipin ito tulad ng isang malakas na shell ng GUI sa `git log` at `git grep`.
+Ito ang tool na gagamitin kapag sinusubukan mong maghanap ng isang bagay na nangyari sa nakaraan, o maisalarawan ang kasaysayan ng iyong proyekto.
 
-Gitk is easiest to invoke from the command-line.
-Just `cd` into a Git repository, and type:
+Ang Gitk ay pinakamadaling tawagin mula sa command-line.
+`cd` lamang sa isang repository ng Git, at i-type:
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: ``Mag-`cd` lamang``

File: book/A-git-in-other-environments/sections/guis.asc [hunk 8]
```diff
@@ -1,112 +1,109 @@
...
 [source,console]
 ----
 $ gitk [git log options]
 ----
 
-Gitk accepts many command-line options, most of which are passed through to the underlying `git log` action.
-Probably one of the most useful is the `--all` flag, which tells gitk to show commits reachable from _any_ ref, not just HEAD.
-Gitk's interface looks like this:
+Tinatanggap ng Gitk ang maraming mga opsyon sa command line, na karamihan ay naipasa sa pinagbabatayan na aksyon ng `git log`.
+Marahil ang isa sa mga pinaka-kapaki-pakinabang ay ang `--all` na flag, na nagsasabi sa gitk na ipakita ang mga commit na mula sa _any_ ref, hindi lamang HEAD.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc:31]: `_anumang_`

File: book/A-git-in-other-environments/sections/guis.asc [hunk 9]
```diff
@@ -1,112 +1,109 @@
...
 
 [source,console]
 -----
 $ git gui
 -----
 
-And it looks something like this:
+At mukhang ganito:
 
 .The `git-gui` commit tool.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc:52]: ``Ang `git-gui` na commit na kasangkapan.``

File: book/A-git-in-other-environments/sections/guis.asc [hunk 10]
```diff
@@ -1,112 +1,109 @@
...
 image::images/git-gui.png[The `git-gui` commit tool.]
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: ``Ang `git-gui` na commit na kasangkapan``

File: book/A-git-in-other-environments/sections/guis.asc [hunk 11]
```diff
@@ -1,112 +1,109 @@
...
-You can also choose to amend the last commit by choosing the ``Amend'' radio button, which will update the ``Staged Changes'' area with the contents of the last commit.
-Then you can simply stage or unstage some changes, alter the commit message, and click ``Commit'' again to replace the old commit with a new one.
+Sa kanang ibaba ay ang mensahe at aksyon ng lawak.
+I-type ang iyong mensahe sa text box at pindotin ang ``Commit'' upang magawa ang isang bagay na katulad ng `git commit`.
+Maaari mo ring piliin na baguhin ang huling commit sa pamamagitan ng pagpili ng radio button na ``Amend'', na kung saan ay i-update ang bahagi ng ``Mga Staged na Pagbabago' sa mga nilalaman ng huling commit.
+Pagkatapos ay maaari mo i-stage o unstage ang ilang mga pagbabago, baguhin ang commit na mensahe, at pindotin muli ang ``Commit' upang palitan ang lumang commit ng bago.
 
-`gitk` and `git-gui` are examples of task-oriented tools.
-Each of them is tailored for a specific purpose (viewing history and creating commits, respectively), and omit the features not necessary for that task.
+`gitk` at `git-gui` ay mga halimbawa ng mga tool na task-oriented.
```
Comment (josteinfrancois) [inline book/A-git-in-other-environments/sections/guis.asc]: - `` Ang `gitk` ``
- You could also change `tool` to `kasangkapan`

**Review by josteinfrancois — COMMENTED** [review]
Hello @dranem ! I have reviewed and commented on your work. Please respond to these comments accordingly. Thanks.

File: book/A-git-in-other-environments/sections/guis.asc (hunk 8 above)
Comment (dranem) [reply book/A-git-in-other-environments/sections/guis.asc]: I didnt changed it to _anumang_ because is a _any_ is a reserved word.

**Pull request merged**